app/controllers/internal_controller.rb

```diff
@@ -1,47 +1,26 @@
 class InternalController < ApplicationController
-  before_action :authenticate
-  skip_before_action :site_wide_authentication
-  helper_method :publisher?
+  helper_method :publisher?, :username
+  before_action :authorize_user
```

Comment:
What's this doing? I would have expected to see something similar to the TTA service, along the lines of:

```
before_action :restrict_access, if: :authenticate?<br class="Apple-interchange-newline">
```